benchees/pocketfft/doit.c

```diff
@@ -0,0 +1,83 @@
+/* this program is in the public domain */
+
+#include "bench-user.h"
+#include "pocketfft.h"
+#include <math.h>
+
+BEGIN_BENCH_DOC
+BENCH_DOC("name", "pocketfft")
+BENCH_DOC("author", "Martin Reinecke")
+BENCH_DOC("year", "2004")
```

Comment:
```suggestion
BENCH_DOC("year", "2018")
```
Latest version is from 2018?

Reply:
```suggestion
BENCH_DOC("year", "2019")
```